database/admin_tool_recommendation.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Can we make this a yaml file. Galaxy prefers yaml so Admins are used to them. Could you also rename this to `tool_recommendations_overwrite.yml.sample` and also maybe include a real example. Cufflinks for example can be deprecated.

Reply:
I can change it to `yml` file. The deprecation is handled by the model itself using `loss` function weighted by the usage frequencies, dot product with the predictions and usage frequencies ... 
The file `admin_tool_recommendation.json` appends recommendations to the ones predicted by the model. It does not overwrite currently. Since overwriting and appending by admins both make sense in different situations, I will add another config parameter to handle it.